.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -15,26 +15,68 @@
 # build from the root of this repo:
 FROM gcr.io/cloud-devrel-public-resources/python
 
-# install tools
+ARG SYNTHTOOL_COMMITTISH=63cc541da2c45fcfca2136c43e638da1fbae174d
+ARG OWLBOT_CLI_COMMITTISH=ac84fa5c423a0069bbce3d2d869c9730c8fdf550
+ENV HOME=/home
+
+# install OS tools
 RUN apt-get update && apt-get install -y \
-	unzip openjdk-17-jdk rsync maven jq \
+	unzip openjdk-17-jdk rsync maven jq less vim \
```

Comment:
I don't see `less` and `vim` being used, I guess the reason we need them is for testing purpose?

Reply:
For local development, I think you can mention them in the development.md. No need to install it in production.

Reply:
Makes sense, I removed these tools from the image and added a small section in the development guide.